src/test/scala/scorex/lagonaki/unit/BlockSpecification.scala

```diff
@@ -13,27 +14,54 @@ import scala.util.Random
 
 class BlockSpecification extends FunSuite with Matchers with MockFactory {
 
-  test(" block with txs bytes/parse roundtrip") {
+  val reference: Array[Byte] = Array.fill(Block.BlockIdLength)(Random.nextInt(100).toByte)
```

Comment:
This has a lot of magic numbers, please rewrite it using generators so that things would be more random